SpatialGDK/Documentation/content/non-unreal-server-worker-types.md

```diff
@@ -121,48 +130,48 @@ Worker_RequestId SendSomeCommandRequest(Worker_EntityId TargetEntityId, Worker_C
 #### Receive data
 You could receive and deserialize a component update and command request in your Unreal project code in the following way:
 ```
-UCLASS()
-class SPATIALGDK_API SessionComponentCallbacks : UOpCallbackTemplate
+void UTPSGameInstance::Init()
 {
-    GENERATED_BODY()
-
-public:
-    Worker_ComponentId GetComponentId()
-    {
-        return 1337;
-    }
-
-    void OnComponentUpdate(const Worker_ComponentUpdateOp& Op) override
-    {
-        Worker_ComponentUpdate Update = {};
-        Update.component_id = 1338;
-        Update.schema_type = Schema_CreateComponentUpdate(1338);
-        Schema_Object* FieldsObject = Schema_GetComponentUpdateFields(Update.schema_type);
-        Schema_AddInt32(FieldsObject, 1, ++UnrealCounter);
-        Cast<USpatialNetDriver>(World->GetNetDriver())->Connection-> SendComponentUpdate(39, &Update);
-    }
-
-    void OnCommandRequest(const Worker_CommandRequestOp& Op) override
-    {
-        Worker_CommandResponse Response = {};
-        Response.component_id = 1338;
-        Response.schema_type = Schema_CreateCommandResponse(1338, 1);
-        Schema_Object* ResponseObject = Schema_GetCommandResponseObject(Response.schema_type);
-
-        const char* Text = "My Response text.";
-        Schema_AddBytes(ResponseObject, 1, (const uint8_t*)Text, sizeof(char) * strlen(Text));
-
-        // Spawn an actor using the World pointer
-        // (where CubeClass is a Blueprint Actor UClass pointer)
-        if (StaticComponentView->HasAuthority(Op.entity_id, GetComponentId()))
+    // OnConnected is an event declared in USpatialGameInstance
+    OnConnected.AddLambda([this]() {
```

Comment:
This seems confusing given the previous instructions. We've told users to register callbacks in Init but then here we are deferring all the callbacks to some later time. The lambda is defined in Init but none of the callbacks are necessarily registered there.
However it would seem that making callbacks in OnConnected should be fine, so we should possibly just tell users to do that.